README.md

```diff
@@ -326,24 +326,28 @@ ReactDOM.render(element, document.getElementById('root'));
 
 In the above example the prop "name" is set as a JSX attribute. React passes all JSX attributes to our user-defined component in a single object.
 
+:trophy: Extend `react-dom/index.js` and `VCompositeNode.js` to handle functional components.
+
 To get functional components working, you should:
 
 1. Extend `instantiateVNode` in `react-dom/index.js` to be able to instantiate a `VCompositeNode`.
    To do this, just check if the `type` attribute of `reactElement` is a `function` (use `typeof`).
 
 You also need to implement `VCompositeNode.js`:
 
-2. The `constructor` need to set the `reactElement`-argument as a class-property.
-3. `mount` should render the `reactElement` class-property by calling `type` as a function with its `props` as the
-   argument (`type(props);`).
-4. Call `instantiateVNode` with the result the rendering we did in step-3 to get a virtual-node.
-5. The last ting we need to do is to call `mount` on the virtual-node we got in step-4 and return the value.
+2. The `constructor` need to set the `reactElement`-argument as a class-property. (Just like we did for `VDomNode` in task 2)
 
-:trophy: Extend `react-dom/index.js` and `VCompositeNode.js` to handle functional components.
+3. The next thing we need to do is to render our component. Call the functional component (`type`) with its `props` as the argument. `this.reactElement.type(this.reactElement.props)` 
```

Comment:
Er det nøvendig å være så eksplisitt? Jeg satt og lurte på det selv da jeg skrev `type(props)`, det er jo bare for illustrasjon, syns egentlig ikke det er vits å gi det med så små t-skjeer? 😅

Reply:
Fjerner det jeg